meta-flex-support/recipes-core/images/production-image.bb

```diff
@@ -1,13 +0,0 @@
-# ---------------------------------------------------------------------------------------------------------------------
-# SPDX-License-Identifier: MIT
-# ---------------------------------------------------------------------------------------------------------------------
-
-SUMMARY = "A production image that fully supports the target device \
-hardware."
-
-IMAGE_FEATURES_PRODUCTION ?= ""
-IMAGE_FEATURES = "${IMAGE_FEATURES_PRODUCTION} ${EXTRA_IMAGE_FEATURES}"
-IMAGE_FEATURES_DISABLED_PRODUCTION ?= "debug-tweaks codebench-debug tools-profile"
-IMAGE_FEATURES:remove = "${IMAGE_FEATURES_DISABLED_PRODUCTION}"
-
-require recipes-core/images/flex-image.inc
```

Comment:
@hashraf91 didn't we discuss merging the inc file in development image recipe, in this commit?

Reply:
This will create conflicts for the second PR. However, I'll make this change as it makes sense to be included in the same commit.